src/java.base/share/native/libjli/manifest_info.h

```diff
@@ -56,6 +56,8 @@
 #define ZIP64_EXTHDR 24       // EXT header size
 #define ZIP64_EXTID   1       // Extra field Zip64 header ID
 
+#define ZIP64_EXTMAXLEN 36 // Maximum Zip64 extra field length
```

Comment:
The fields described in APPNOTE-6.3.9.TXT 4.5.3 are total 32 bytes. Any other additional fields in the Zip64 extended information?

```
        Value      Size       Description
        -----      ----       -----------
(ZIP64) 0x0001     2 bytes    Tag for this "extra" block type
        Size       2 bytes    Size of this "extra" block
        Original 
        Size       8 bytes    Original uncompressed file size
        Compressed
        Size       8 bytes    Size of compressed data
        Relative Header
        Offset     8 bytes    Offset of local header record
        Disk Start
        Number     4 bytes    Number of the disk on which
                              this file starts 
```

Reply:
Thanks for the catch, I had missed that the disk start number is 4 bytes and not 8. I pushed a commit. I also removed some unused references to the disk number, which is only being used to validate the size of the zip64 extended info.

Reply:
Ok. Could you please also fix the above `36`? Please change the start of `//` (comment) to be aligned with the earlier comment lines.

Reply:
Done, thanks